bin/admin/check_uid_gid_collisions.pl

```diff
@@ -0,0 +1,328 @@
+#! /usr/bin/env perl
+# vim: set filetype=perl ts=4 sw=4 sts=4 et:
+use strict;
+use warnings;
+use 5.010;
+
+use Getopt::Long;
+
+use File::Basename;
+use lib dirname(__FILE__) . '/../../lib/perl';
+
+my $getOptionsOk = GetOptions(
+    "master-passwd=s" => \my $masterPasswdFile,
+    "local-passwd=s"  => \my $localPasswdFile,
+    "master-group=s"  => \my $masterGroupFile,
+    "local-group=s"   => \my $localGroupFile,
+    "output=s"        => \my $outputFile,
+    "offset=i"        => \my $offset,             # undocumented on purpose
+);
+
+if (!$getOptionsOk) {
+    print "Failed to parse options (see above), aborting.\n";
+    exit(1);
+}
+
+$localPasswdFile //= '/etc/passwd';
+$localGroupFile  //= '/etc/group';
+$offset          //= 50_000_000;
+
+if ($offset < 10_000) {
+    die "Offset is too low ($offset)";
```

Comment:
Add `\n` on your `die`s so it does not output a stack trace

Reply:
Done on this one and the others too. Also replaced an `exit(1)` with a `die`, to be more consistent.